fuzz.go

```diff
@@ -332,6 +332,11 @@ func (fc *fuzzerContext) doFuzz(v reflect.Value, flags uint64) {
 	case reflect.Interface:
 		fallthrough
 	default:
+		for _, pattern := range fc.fuzzer.skipFieldPatterns {
+			if pattern.MatchString(v.Type().Name()) {
```

Comment:
This looks to be matching the type name, not the field name. I suspect the test passes even without this change?